.circleci/config.yml

```diff
@@ -248,7 +248,7 @@ references:
       environment:
         PACKAGES_DIR: *packages_workspace
       command: |
-        ./deployment/upload-github-release-assets.sh github_api_token=${GITHUB_API_TOKEN:?} \
+        /infrastructure/scripts/upload-github-release-assets.sh github_api_token=${GITHUB_API_TOKEN:?} \
```

Comment:
I see in [2.0.1 tag](https://github.com/aeternity/infrastructure/blob/v2.0.1/scripts/upload-github-release-assets.sh), in [stable](https://github.com/aeternity/infrastructure/blob/151109b848aa4cabb037482c62e8570c235f7fb2/scripts/upload-github-release-assets.sh), in [master](https://github.com/aeternity/infrastructure/blob/f347c0b6af0e485f245b44b88ce6c8cb5eef39a7/scripts/upload-github-release-assets.sh).